sections/combat.tex

```diff
@@ -21,7 +21,7 @@ \subsection*{\hypertarget{Combatsetup}{Combat Setup}}
 \begin{itemize}
   \item Choose one of the Combat Board's sides as your own.
     Place up to 5 of your Unit cards freely onto the Back and Frontlines of that side.
-  \item Check the \hyperlink{Difficulty}{\textbf{Difficulty Table}} and draw the corresponding number of Neutral Unit cards from their Decks.
+  \item Check the \hyperlink{Difficulty Table}{\textbf{Difficulty Table}} and draw the corresponding number of Neutral Unit cards from their Decks.
```

Comment:
In the printable build, there will be page 52 here 😆 You think that's OK?

Reply:
I'm open for suggestions

Reply:
I mean, the printable version will say explicitly (p. 52), but there is no such page. I guess we should exclude this from the Python script.

Reply:
PTAL. In future we should choose which link should add (p. N) and which should not, and remove the script.